test/e2e/README.md

```diff
@@ -192,6 +192,35 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster and push antrea on it, the following steps should be done for running the e2e tests or other kinds of tests.
+
+You should create the `config` file under the default ssh directory and add the ssh information of each nodes in the cluster to it. You can add the ssh information like the example below:
+
+```bash
+Host Control-Plane-Node
+    HostName 127.0.0.1
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host Worker-Node
+    HostName <NodeIP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
+The example above is to launch the ssh connection from the `Control-Plane-Node` to itself and the `Worker-Node` when launching the tests on `Control-Plane-Node`. You should make sure `Host` of each node must be the same as the corresponding node name. The `Port` is the default port of ssh service. As for the `IdentityFile`, add the location of the private key (`id_rsa` in example) to it.
+
+Except the configuration of the ssh connection, make sure you have added the public key to `authorized_keys` of each nodes and set `PubkeyAuthentication` and `RSAAuthentication` of ssh service to be `yes`. 
```

Comment:
change `Except` to `Beside`, `each nodes` to `each node`,  'to be `yes`' to 'to `yes`'